src/plugins/workspace/server/saved_objects/workspace_saved_objects_client_wrapper.ts

```diff
@@ -66,8 +70,32 @@ const getDefaultValuesForEmpty = <T>(values: T[] | undefined, defaultValues: T[]
   return !values || values.length === 0 ? defaultValues : values;
 };
 
+export const isRequestByDashboardAdmin = (
```

Comment:
Prefer move to some utils file

Reply:
Have moved to `src/plugins/workspace/server/utils.ts` and add unit test.